programs/client/CMakeLists.txt

```diff
@@ -1,6 +1,22 @@
+PROTOBUF_GENERATE_CPP(GRAMMAR_SRC GRAMMAR_HDR buzz-house/ast/sql_grammar.proto)
```

Comment:
It's used by other components from CH. You can search for it.

Reply:
Got it, I had to reconfigure with `ENABLE_PROTOBUF=1`.
If we're going to always require protobuf for the client, I suggest we drop a nice message to help us figure out what to enable and how.

We do this sort of thing in other places: https://github.com/ClickHouse/ClickHouse/blob/9f62518846ca945c11a653b46d2d3f3885ad7f40/contrib/aws-cmake/CMakeLists.txt#L9-L15

Reply:
I guess I should introduce a new option to compile BuzzHouse or not

Reply:
That'd be cool if it can be enabled/disabled on demand, but at least it's always helpful to know what needs to be configured in CMake to make something work. Searching for internal dependencies when not stated explicitly is not as easy as a direct message.